seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/resources/kafka/kafka_to_kafka_exactly_once.conf

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+env {
+    execution.parallelism = 1
+    job.mode = "STREAMING"
+  }
+
+source {
+  Kafka {
+    bootstrap.servers = "kafkaCluster:9092"
+    topic = "kafka_topic_exactly_once"
+    result_table_name = "kafka_topic_exactly_once"
+    # The default format is json, which is optional
+    format = json
+    start_mode = earliest
+  }
+
+}
+transform {}
+
+
+sink{
+  kafka {
+        format = JSON
+        topic = "kafka_topic_exactly_once"
```

Comment:
Distinguish between different topics

Reply:
It has been modified, but after running CI Kafka Connect a few times, there is still a problem. I haven't found a solution yet
![image](https://github.com/user-attachments/assets/7049ab55-2b18-4343-89bc-e31565135a6d)

Reply:
![image](https://github.com/user-attachments/assets/7f3a4e7e-083c-4ee7-8d39-ba7212189394)